corehq/apps/sms/forms.py

```diff
@@ -378,7 +379,11 @@ def section_registration(self):
                     "configured in the SMS languages and translations page "
                     "(Messaging -> Languages -> Messaging Translations)."),
             ),
-            hqcrispy.FieldWithHelpBubble(   # TODO: hide if project only has one language
```

Comment:
why is this not relevant in case of a single language? the case to avoid the error in case of no backup would still be applicable? For ex: if ICDS had only one language today?

Reply:
At least one translation is required, so if there's only one language there ought to be a translation. Although, there are edge cases around adding and removing languages, so...yeah, I can add this back, removing it doesn't have much benefit.

Reply:
Cool

Reply:
updated in 626337c580e